deploy/docker-compose/docker-compose.yml

```diff
@@ -76,28 +76,29 @@ services:
     networks:
       - fba_network
 
-#  # For use fastapi_best_architecture_ui, or online/pro env
-#  fba_ui:
-#    build:
-#      context: ⚠️ your fba_ui folder directory ⚠️
-#      dockerfile: Dockerfile
-#    ports:
-#      - "80:80"
-#      - "443:443"
-#    container_name: fba_ui
-#    restart: always
-#    depends_on:
-#      - fba_server
-#    command:
-#      - nginx
-#      - -g
-#      - daemon off;
-#    volumes:
-#      - fba_static:/www/fba_server/backend/app/static
-#    networks:
-#      - fba_network
+  #  # If you choose to use fastapi_best_architecture_ui (fba_ui), you can open this annotation
```

Comment:
Whether to transfer to the front-end project

Reply:
Maybe, I'll try.